manual_controller_node:
  ros__parameters:
    manual_controller:
      topic_name:
        cmd_vel: "manual/cmd_vel"
        auto_cmd_vel: "auto/cmd_vel"
        joy: "/joy"
      max:
        vel: 0.3 #[m/s]
        angular: 1.0 #[rad/s]